### PR TITLE
Parent attribute support

### PR DESCRIPTION
**Pull request by int-index**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+resolver: lts-6.27
+
+packages:
+- "."
+- location:
+    git: https://github.com/typeable/dom-parser
+    commit: 0a8ba3a6b5f5ee063d235e287997db7cf12e79ee
```
Comment (int-index) [inline stack.yaml]: reminder: upload the new dom-parser to Hackage

Comment (s9gf4ult) [reply stack.yaml]: You said you will merge PR in dom-parser, so I dont touch it. When you merge your PR I will upload new version.

Comment (int-index) [reply stack.yaml]: OK, I merged it.

File: src/Text/XML/ParentAttributes.hs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import Data.Text.Lazy as T.L
+import Data.Text.Lazy.Builder as T.L
+import Data.Text.Lazy.Builder.Int as T.L
+import Numeric.Natural
+import Text.XML
+
+class ToXmlParentAttributes a where
+  toXmlParentAttributes :: a -> [(Name, T.S.Text)]
+
+instance {-# OVERLAPPABLE #-} ToXmlParentAttributes a where
```
Comment (s9gf4ult) [inline src/Text/XML/ParentAttributes.hs]: Document this instance plz

Comment (int-index) [reply src/Text/XML/ParentAttributes.hs]: I wrote a comment that describes the motivation behind this instance and how it's used, with an example. Does it help?

Comment (s9gf4ult) [reply src/Text/XML/ParentAttributes.hs]: Yes.

**Review by s9gf4ult — APPROVED** [review]
New dom-parser is uploaded.

**Pull request merged**